README.md

```diff
@@ -1 +1,101 @@
-Hi
+# Kayaks Next
+
+This webpage serves to promote, show and market the Kayaks produced in La Isla.
+
+## Built With 
+
+- HTML/CSS
+- NextJS
+- Node.JS
+
+
+## Getting Started
+
+If you want a copy of this file go to the github repository and download it from there:
+- [`https://github.com/Metsanpeitto/kayaks_next`](https://github.com/Metsanpeitto/kayaks_next)
+
+Project Kanban Board:
+- [`https://github.com/Metsanpeitto/kayaks_next`](https://github.com/Metsanpeitto/kayaks_next)
+
+
+
+### Live
+
+
+### Setup
+
+```cmd
+git clone  https://github.com/Metsanpeitto/kayaks_next.git
+cd ./kayaks_next
+```
+
+### Install
+
+```cmd
+npm install
+```
+
+### Usage
+
+```cmd
+npm start
+```
+
+### Verify linters
+
+- Autofix linters errors
+
+```cmd
+npx eslint . --fix
+npx stylelint "**/*.{css,scss}" --fix
+```
+
+## To run jest test
+```cmd
+npm run test
+npm run jest
```

Comment:
Two? difference?

Reply:
They aren't really needed for now. 
i remove both.